packages/core/src/config/types.ts

```diff
@@ -148,6 +148,7 @@ export type SphinxConfig = {
 export interface RawCreate2ActionInput extends SphinxTransaction {
   contractName: string | null
   create2Address: string
+  initCodeWithArgs: string
```

Comment:
Added this field to the `RawCreate2ActionInput` type because it provides a convenient way to aggregate all of the init code, which we'll pass into `getConfigArtifacts`